Documentation/Metrics/arangodb_heartbeat_failures_total.yaml

```
@@ -9,8 +9,13 @@ complexity: simple
 exposedBy:
   - coordinator
   - dbserver
+  - single
 description: |
   Total number of failed heartbeat transmissions.
+  Servers in a cluster periodically send their heartbeats to
+  the agency to report their own liveliness. This counter gets
+  increased whenever sending such heartbeat fails. In the single
```

"such heartbeat" -> "such a heartbeat"
```suggestion
  increased whenever sending such a heartbeat fails. In the single
```